src/app.vue

```diff
@@ -16,19 +16,23 @@ export default {
   components: { ModuleToast, ModuleDecisionTree },
   mounted() {
     this.setOnlineStatus(window.navigator.onLine);
+    if (window.navigator.onLine) {
+      this.setDarkList();
+      this.setCurrency('USD');
```

Comment:
can you import 'USD' from a currency object. Lets initially ad USD, EUR, CHR, SGD, RUB AND RMB

Reply:
done!